frontend/packages/kubevirt-plugin/src/selectors/pod/selectors.ts

```diff
@@ -47,26 +46,29 @@ export const isPodSchedulable = (pod: PodKind) => {
   );
 };
 
+const isPodReady = (pod: PodKind): boolean =>
+  pod?.status?.phase === 'Running' && pod?.status?.containerStatuses?.every((s) => s.ready);
+
 export const findVMPod = (
```

Comment:
why not changing the name to `findVMIPod` too ?

Reply:
Thansk ! fixing

Reply:
done :+1: